db/migrations/02_create_peeps_table.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE peeps (id SERIAL PRIMARY KEY, text VARCHAR(280))
+ALTER TABLE "peeps" ADD COLUMN "user_id" varchar(60);
```

Comment:
I would have institiated directly as foreign key in the schema of that table. So the relationship between user and peep is clear from a db perspective.